cmd/gardenlet/app/app.go

```diff
@@ -274,6 +274,9 @@ func (g *garden) Start(ctx context.Context) error {
 				&gardencorev1beta1.ControllerInstallation{}: {
 					Field: fields.SelectorFromSet(fields.Set{gardencore.SeedRefName: g.config.SeedConfig.SeedTemplate.Name}),
 				},
+				&gardencorev1beta1.Seed{}: {
+					Field: fields.SelectorFromSet(fields.Set{metav1.ObjectNameField: g.config.SeedConfig.SeedTemplate.Name}),
+				},
```

Comment:
I think this restriction causes issues with the `ManagedSeed` controller - see https://github.com/gardener/gardener/blob/ba85cef85a1d029c3f6ed84aea2753f57bc07d62/pkg/gardenlet/controller/managedseed/actuator.go#L171.